src/aks-preview/azext_aks_preview/machine.py

```diff
@@ -0,0 +1,130 @@
+from azext_aks_preview._client_factory import CUSTOM_MGMT_AKS_PREVIEW
```

Comment:
Please add license header (see examples from other files) to this newly added file.